Sources/SUIComponents/Atoms/Button+HMRC.swift

```diff
@@ -18,7 +18,7 @@ import SwiftUI
 
 public enum HMRCButtonStyle {
     case primary
-    case secondary(padding: CGFloat = .spacer16)
+    case secondary(padding: CGFloat = .spacer16, icon: Image? = nil)
```

Comment:
As discussed - lets lose this until we make the [IconButtonView Component](https://jira.tools.tax.service.gov.uk/browse/HMA-4860)